src/js/components/Calendar/Calendar.js

```diff
@@ -254,6 +261,8 @@ const Calendar = forwardRef(
       };
     }, []);
 
+    const [normalize, setNormalize] = useState(normalizeProp);
```

Comment:
Can we add a comment above this briefly explaining what `normalize` means and how it is used?

Reply:
Done.